docs/release_notes.md

```diff
@@ -1,5 +1,97 @@
+
+The biggest news of this release is that `imiscsvrinfo` has been updated to report SSL/TLS certificate information, if in use. This is made possible due to enhancements to the `rcGetMiscSvrInfo` API endpoint. Keep in mind that this requires the client to have a proper client-side configuration. 
+
+This release consists of [94 commits from 6 contributors](https://github.com/irods/irods/compare/4.3.3...4.3.4) and [closed 134 issues marked for 4.3.4](https://github.com/irods/irods/issues?q=milestone%3A4.3.4).
+
+The latest binary packages for AlmaLinux8, RockyLinux9, Ubuntu20, Ubuntu22, Ubuntu24, Debian11, and Debian12 are available at <https://packages.irods.org/>.
+
+### Changed
+
+- Improve documentation (#2600, #7622, #7701, #7784, #7952, #8000).
+- Improve testing (#6421, #7412, #7491, #7795, #7990, #8046, #8047, #8183, #8185, #8247).
+- Clean up code (#5800, #6972).
+- Remove unnecessary code - source files, functions, declarations, header includes, etc (#6043, #6234, #6546, #7919, #7942, #8010, #8133, #8201, #8204).
+- Replace use of externals-provided libarchive with distro-provided libarchive (#6250, #7286).
+- Improve CMake (#6319, #6584).
+- Update clang-format / clang-tidy configuration (#6970, #7313, #7751, #7821, #7822).
+- Remove ACLs following removal of user (#7778).
+- Migrate GitHub workflows to Ubuntu 24.04 (#7823).
+- Expand permission levels supported by atomic ACL operations API (#7913).
+- Reduce log noise (#7953).
+- Remove installation of packedRei directory from CMakeLists.txt (#7993).
+- Install msiExecCmd_bin/hello script as a template (#7994).
+- Replace stacktrace with user-friendly error message in `itree` (#8082).
+- Improve GenQuery2 parser's handling of whitespace sequences (#8182).
+- Add json_events.hpp header file to development package (#8200).
+
+### Removed
+
+- Remove CentOS 7 build from GitHub workflows (#7968).
+
+### Deprecated
+
+- Deprecate `ilocate` (#2524).
+- Deprecate `igetwild` (#2525).
+- Deprecate modification of user having rodsgroup type in GeneralAdmin API (#2978).     THIS ISSUE IS REFERENCED IN TWO SECTIONS. IT DOES COVER TWO THINGS.
+- Deprecate modification of ACL policy (#6843).
+- Deprecate GeneralRowInsert and GeneralRowPurge (#7608).
+- Deprecate update_deprecated_database_columns.py (#7835).
+- Deprecate `imeta qu` (#7959).
+- Deprecate `imeta` interactive mode (#7961).
+- Deprecate server monitoring microservices (#7977).
+- Deprecate DataObjLock and DataObjUnlock (#7979).
+- Deprecate unused global variables in resource manager implementation. (#8042).
+- Deprecate `fillGenQueryInpFromStrCond` (#8088).
+- Deprecate `forkAndExec` (#8109).
+
+### Fixed
+
+- Avoid SIGABRT by setting pointers to null after deallocation (#3581).
+- Return errors on one-sided encryption in parallel transfer (#4984).       THINK ABOUT THIS ONE. ISSUE IS OPEN, BUT HAS BEEN BUMPED TO 5.0 MILESTONE.
```

Comment:
I'm wondering if issue 4984 should be closed instead of being moved into the 5.0.0 milestone.

Hmm.

Reply:
'Changed' does sound a bit more correct to me.

It is definitely cleaner to have work done in one release be closed during that release... but there is precedent in us having commits in the system that do NOT yet close an issue.   However, I don't think we've ever put issue numbers in the release notes that weren't closed.   So... we're in new territory either way.

Smaller/discrete issues are the real answer.

Reply:
I agree with both comments.

Will move it to the "Changed" section.

I think it's important that we mention the change because it is a change in behavior. I'm going to leave it in even though the release notes normally only reference closed issues. The issue just gives the reader a place to start understanding the story.

Reply:
Moved to "Changed" section.